clients/csharp-dotnet-core/voice-assistant-test/tool/BotConnector.cs

```diff
@@ -600,11 +611,13 @@ private void SpeechBotConnector_Canceled(object sender, SpeechRecognitionCancele
 
         private void SpeechBotConnector_SessionStarted(object sender, SessionEventArgs e)
         {
+            this.stopWatch.Start();

```

Comment:
The code already has start/stop watch for caluclating latency. You need to adjust them, not create new ones. Let me look at the full code.